apps/monarch/monarch-frontend/src/app/ManageTherapists.tsx

```diff
@@ -27,8 +24,33 @@ import { controller } from './actionsController';
 import { PractitionerInfo } from '@c4c/monarch/common';
 Amplify.configure(awsmobile);
 
-const ManageTherapists: React.FC<{ accessToken: string, reload: boolean, setReload: (arg: boolean) => void }> = ({accessToken, reload, setReload}) => {
+const renderBadges = (therapist: PractitionerInfo) => {
+  const badgeList = [
+    ...therapist.languagesList.map((language) => ({
+      label: language,
+      colorScheme: 'orange',
+    })),
+  ];
 
+  badgeList.push({ label: 'New', colorScheme: 'teal' });
```

Comment:
IMO it doesn't make that much sense to display "New" for a pending practitioner since they haven't joined yet - what do you think?